### DISCUSSION TITLE
unexpected output from example4_spm

### DISCUSSION THREAD

**Discussion by HongUKA** [cvnlab/GLMsingle#118] [Q&A]
Hi, I am new to GLMSingle. I ran yesterday the script example4_spm with matlab. There is no error message. But the betas of TYPED_FITHRF_GLMDENOISE_RR.mat(modelmd) look different than the result of example1. This looks like noise. what should I do, so that I can get a result like example1? Many thanks for your help!

**Comment by kendrickkay** [comment]
Hi, we will need more information. Can you post a screenshot and/or a log
of what commands you ran?

**Reply by HongUKA** [reply]
Thanks for your quick response. This example code example4_SPM.m is in folder GLMsingle/matlab/examples/. This screenshot ist one beta  from TYPED_FITHRF_GLMDENOISE_RR.mat (modelmd). 
![screenshot](https://github.com/cvnlab/GLMsingle/assets/151827704/2fa254d1-58f3-46c1-8bfc-827d7ee4f195)

**Comment by kendrickkay** [comment]
This looks like it might be correct. The units are %BOLD.  Try a range like -5 to 5.   The extreme values outside the brain are just noise that results from dividing by small numbers (for the mean signal intensity).

**Comment by HongUKA** [comment]
Thank you very much for the explanation
